scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatWriter.scala

```diff
@@ -622,6 +647,28 @@ class FormatWriter(formatOps: FormatOps) {
     }
   }
 
+  private def prepareAlignmentInfo(
+      block: Vector[Array[FormatLocation]],
+      separatorLengthGaps: Array[Int],
+      column: Int
+  ): Vector[AlignmentUnit] =
```

Comment:
very, very optional: why are we using `Vector` when we don't do index lookups? http://www.lihaoyi.com/post/BenchmarkingScalaCollections.html#lists-vs-vectors

Reply:
Because `Vector` used here before 🤷‍♂ 

May be it comes because of append operation [here](https://github.com/scalameta/scalafmt/blob/master/scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatWriter.scala#L585)